packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -147,6 +147,30 @@ var alreadyValidDateTime = function (dateTimeString) {
   );
 };
 
+const incompeteIso8601Regex = /^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}$/;
+const Iso8601RegexWithMissingT = /^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}(\.\d{3}Z)$/;
+
+var incompleteIso8601 = function (dateTimeString) {
```

Comment:
Why not use the regular way of declaring functions?

```
function incompleteIso8601(dateTimeString) {
...
```

Reply:
I understand, but if not a blocker, I'd code it using our standards - we can later discuss about older code and whether to update it or not.

Not a blocker for this one now, since we need to push it sooner than later, but let's keep that in mind, please.